docs/nydus-image.md

```diff
@@ -1,65 +1,187 @@
-# Nydus Image Builder
+# Toolset for Working with RAFS Filesystems and Nydus Container Images
 
-`nydus-image` tool converts a directory tree (usually an image layer) into two parts: `bootstrap` and `blob`:
+The `nydus-image` toolset provides tools to build, check, inspect and export RAFS filesystems and Nydus container images.
 
-- `bootstrap` is a file presenting filesystem metadata information of the directory;
-- `blob` stores all files data in the directory;
+Logically, a RAFS filesystem consists of two parts:
+- a bootstrap/meta blob, containing filesystem metadata, such directory name, file attributes etc.
+- one or more data blobs, containing file contents.
 
-### Get binary from release page
+Physically, RAFS bootstrap/meta blobs can be stored as separate files, or inlined into data blob files.
+Therefore, a RAFS file system may consist of the following parts:
+- a blob file containing both RAFS metadata and data
+- a blob file containing RAFS metadata, and one or more blob files for RAFS data
+- a blob file containing RAFS metadata, and one or more blob files for RAFS data, and associated targz files for RAFS ZRAN mode.
 
-Get `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
+## Installation
 
-## Build Nydus Image From Directory Source
+Get latest `nydus-image` binary from [release](https://github.com/dragonflyoss/image-service/releases/latest) page.
 
+## Nydus Image Builder
+
+The `nydus-image create` subcommand creates a RAFS filesystem/Nydus image from a tar file or from a directory.
```

Comment:
How about "creates a layer or blob of RAFS filesystem/Nydus image"

Reply:
fixed